src/routes/Dashboard/SummaryCard/index.jsx

```diff
@@ -1,26 +1,45 @@
 import {useState} from "react";
 import {BiTime, BiDollarCircle} from "react-icons/bi";
 import {DateTime} from "luxon";
+import {useMutation} from "@apollo/client";
 import Button from "components/main/Button";
 import SummaryCardSection from "routes/Dashboard/SummaryCard/components/SummaryCardSection";
 import CounterTime from "routes/Dashboard/SummaryCard/components/CounterTime";
 import DeleModal from "components/modals/DeleModal";
+import PropTypes from "prop-types";
+import CreateCheckInMutation from "mutations/CreateCheckInMutation";
 
-export default function SummaryCard() {
+export default function SummaryCard({vehicleId}) {
     const [showModal, setShowModal] = useState(false)
     const [counterIsActive, setCounterIsActive] = useState(false)
+    const [dateTime, setDateTime] = useState(null)
 
-    const datetime = "2022-10-20T15:25:00"
     const closeModal = () => setShowModal(false)
 
+    const [createCheckIn] = useMutation(CreateCheckInMutation, {
+        onCompleted: (data) => {
+            setDateTime(data.createCheckIn.data.attributes.createdAt)
+        }
+    })
+
     const handleClick = () => {
-        if (counterIsActive){
+        if (counterIsActive) {
             const now = DateTime.now()
-            const checkInDate = DateTime.fromISO(datetime)
+            const checkInDate = DateTime.fromISO(dateTime)
             const {minutes} = now.diff(checkInDate, ["minutes"]).toObject()
             if (minutes < 5)
                 setShowModal(true)
-        } else setCounterIsActive(true)
+        } else {
+            createCheckIn({
+                variables: {
+                    data: {
+                        parking_lot: 88,
+                        vehicle: vehicleId
+                    }
+                }
+            })
+            setCounterIsActive(true)
```

Comment:
you won't need the `counterIsActive` variable anymore, as you're now using the `dateTime` from the state, which you're setting after the mutation is completed.